### PR TITLE
Site : adding a new field information + a markdown editor.

### PR DESCRIPTION
**Pull request by RamBoFe**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,32 +15,33 @@
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.6.3",
     "@nuxtjs/auth": "^4.5.3",
-    "@nuxtjs/axios": "^5.4.1",
-    "@nuxtjs/pwa": "^3.0.0-beta.12",
+    "@nuxtjs/axios": "^5.0.0",
```
Comment (Nainterceptor) [inline package.json]: Why downgrades ?

File: pages/campuses/_id.vue [hunk 2]
```diff
@@ -5,13 +5,10 @@
 </template>
 
 <script>
-const EDITABLE_FIELDS = 'id,name,location,phone(drivers,everybody),categories(id,label)';
+const EDITABLE_FIELDS = 'id,name,location,phone(drivers,everybody),categories(id,label),informations';
```
Comment (Nainterceptor) [inline pages/campuses/_id.vue]: information, in english, is non-plural, so it's information

File: pages/campuses/_id.vue [hunk 3]
```diff
@@ -5,13 +5,10 @@
...
 
 export default {
-  async asyncData({ app, params, $api }) {
-    if (!app.$auth.hasRight('canGetCampus')) {
-      throw new Error('Vous n\'avez pas les droits pour récupérer les informations d\'une base.');
-    }
+  async asyncData({ params, $api }) {
```
Comment (Nainterceptor) [inline pages/campuses/_id.vue]: Merging issue I guess, why this part has been removed ?

File: pages/campuses/_id/edit.vue [hunk 4]
```diff
@@ -116,13 +129,15 @@
 import ecField from '~/components/form/field.vue';
 import ecGpsPoint from '~/components/form/gps-point.vue';
 import searchCategories from '~/components/form/search-categories.vue';
+import markdownEditor from 'vue-simplemde/src/markdown-editor';
 
-const EDITABLE_FIELDS = 'id,name,location,phone(drivers,everybody),categories(id,label)';
+const EDITABLE_FIELDS = 'id,name,location,phone(drivers,everybody),categories(id,label),informations';
```
Comment (Nainterceptor) [inline pages/campuses/_id/edit.vue]: same thing about information

File: package.json (hunk 1 above)
Comment (RamBoFe) [reply package.json]: Fix on 939c7e8a8dfca0995ee683f2a8bcfe91a88377b7

File: pages/campuses/_id.vue (hunk 2 above)
Comment (RamBoFe) [reply pages/campuses/_id.vue]: Fix on 939c7e8a8dfca0995ee683f2a8bcfe91a88377b7

File: pages/campuses/_id.vue (hunk 3 above)
Comment (RamBoFe) [reply pages/campuses/_id.vue]: Fix on 939c7e8a8dfca0995ee683f2a8bcfe91a88377b7

File: pages/campuses/_id/edit.vue (hunk 4 above)
Comment (RamBoFe) [reply pages/campuses/_id/edit.vue]: Fix on 939c7e8a8dfca0995ee683f2a8bcfe91a88377b7

**Pull request merged**